src/botPage/view/blockly/blocks/tools/time/todatetime.js

```diff
@@ -0,0 +1,35 @@
+// https://blockly-demo.appspot.com/static/demos/blockfactory/index.html#wupwb4
+import { translate } from '../../../../../../common/i18n';
+// import { disable } from '../../../utils';
+
+Blockly.Blocks.todatetime = {
+    init: function init() {
+        this.appendDummyInput();
+        this.appendValueInput('TIMESTAMP').appendField('To Date/Time');
```

Comment:
Make sure to run this through the `translate` function:

```suggestion
        this.appendValueInput('TIMESTAMP').appendField(translate('To Date/Time'));
```